### PR TITLE
Bump to 1.4.15

### PR DESCRIPTION
**Pull request by stephdl**
https://roundcube.net/news/2023/10/16/security-updates-1.5.5-and-1.4.15

move to security update

NethServer/dev#6767

### CODE REVIEW

File: roundcubemail.spec [hunk 1]
```diff
@@ -1,6 +1,6 @@
-%define rcm_version 1.4.13
+%define rcm_version 1.4.15
 Name:           roundcubemail
-Version: 1.4.13
+Version: 1.4.15
```
Comment (gsanchietti) [inline roundcubemail.spec]: Please revert the change to this line, we are going to bump the release after the merge

Comment (stephdl) [reply roundcubemail.spec]: yep

**Review by gsanchietti — APPROVED** [review]
you can merge it, but squash the last commit inside the first one
